src/main/java/core/basesyntax/service/ReadFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReadFromFile {
```

Comment:
```suggestion
public interface FileReader {
```

Reply:
I'm really try to this. But sorry, it's impossible. FileReader is inner system class or someone else for Java. IDEA failed all my tries to do this rename.
I'm rename ReadFromFile to FileReaderService